### ISSUE TITLE
The application is not compatible with Huawei devices

### ISSUE THREAD

**Issue by Lucas-lyh** [thanksmister/wallpanel-android#239]
When open dashboard on Huawei devices, it is completly blank. Is this bug or my fault?

Huawei devices only use "Huawei webview", I guess this is why nothing is shown in dashboard. 
If it is true, Is there any plan to support these devices?

**Comment by Jopinder** [comment]
I'm currently using it on a Huawai MatePad T8 (kob2-w09). Sideloaded the apk since this device is without GMS and it works just fine.

**Comment by Lucas-lyh** [comment]
Yeah, I find it works fine on a Huawei MatePad pro too. Maybe it can not works properly only on Mate40-series?

**Comment by av1024** [comment]
Looks like a webview issue.
Have similar issue on AOSP android with "custom" webview. Solved by installing Google WebView from apkmirror

**Comment by thanksmister** [comment]
I would say Huawei devices are not compatible with Android WebView or offer bad support for the WebView component.  This is not an issue the application can resolve, thi is an issue with the device manufacturer.   You should post a bug on Huawei's site or with Google AOSP.
